src/ralph/security/models.py

```diff
@@ -0,0 +1,128 @@
+#    _ = Choices.Choice
+#
+#    SUPPORT = Choices.Group(0)
+#    new = _("new")
+#
+#
+#class Support(
+#    Regionalizable,
+#    PermByFieldMixin,
+#    NamedMixin.NonUnique,
+#    TimeStampMixin,
+#    TaggableMixin,
+#    models.Model,
+#):
+#    contract_id = models.CharField(max_length=50, blank=False)
+#    description = models.CharField(max_length=100, blank=True)
+#    # TODO: new attachment mechanism
+#    # attachments = models.ManyToManyField(
+#    #     models_assets.Attachment, null=True, blank=True
+#    # )
+#    price = models.DecimalField(
+#        max_digits=10, decimal_places=2, default=0, null=True, blank=True,
+#    )
+#    date_from = models.DateField(null=True, blank=True)
+#    date_to = models.DateField(null=False, blank=False)
+#    escalation_path = models.CharField(max_length=200, blank=True)
+#    contract_terms = models.CharField(max_length=200, blank=True)
+#    remarks = models.TextField(blank=True)
+#    sla_type = models.CharField(max_length=200, blank=True)
+#    # asset_type = models.PositiveSmallIntegerField(
+#    #     choices=AssetType()
+#    # )
+#    status = models.PositiveSmallIntegerField(
+#        default=SupportStatus.new.id,
+#        verbose_name=_("status"),
+#        choices=SupportStatus(),
+#        null=False,
+#        blank=False,
+#    )
+#    producer = models.CharField(max_length=100, blank=True)
+#    supplier = models.CharField(max_length=100, blank=True)
+#    serial_no = models.CharField(max_length=100, blank=True)
+#    invoice_no = models.CharField(max_length=100, blank=True, db_index=True)
+#    invoice_date = models.DateField(
+#        null=True, blank=True, verbose_name=_('Invoice date'),
+#    )
+#    period_in_months = models.IntegerField(null=True, blank=True)
+#    # property_of = models.ForeignKey(
+#    #     AssetOwner,
+#    #     on_delete=models.PROTECT,
+#    #     null=True,
+#    #     blank=True,
+#    # )
+#    support_type = models.ForeignKey(
+#        SupportType,
+#        on_delete=models.PROTECT,
+#        blank=True,
+#        default=None,
+#        null=True,
+#    )
+#    base_objects = models.ManyToManyField(BaseObject, related_name='supports')
+#
+#    def __init__(self, *args, **kwargs):
+#        self.saving_user = None
+#        super(Support, self).__init__(*args, **kwargs)
+#
+#    def get_natural_end_support(self):
+#        return naturaltime(datetime(*(self.date_to.timetuple()[:6])))
+
+
+
+from django.db import models
+
+class SecurityScan(
+    #Regionalizable,
+    PermByFieldMixin,
+    #NamedMixin.NonUnique,
+    TimeStampMixin,
+    TaggableMixin,
+    models.Model,
+):
+    last_scan_date = models.DateTimeField()
+    scan_status = models.PositiveIntegerField(choices=ScanStatus())
+    next_scan_date = models.DateTimeField()
+    details_url = models.URLField(max_length=255, blank=True)
+    rescan_url = models.URLField(blank=True, verbose_name='Rescan url')
+    asset = models.OneToOneField(BaseObject, primary_key=True)
```

Comment:
to think about if it should be OneToOneField or ForeignKey (ex. what if one asset has multiple ip addresses)?